homeassistant/components/rfxtrx/__init__.py

```diff
@@ -268,7 +300,7 @@ def get_device_id(device, data_bits=None):
 
 def fire_command_event(hass, entity_id, command):
```

Comment:
Is this called from async ? If so, let's mark it as `@callback` and prefix name with `async_`

Reply:
Function will go away here: https://github.com/home-assistant/core/pull/37565